ci/build_wheel.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION
+
+
+rapids-logger "Install CUDA Toolkit"
+yum update
+yum install -y epel-release
+yum install -y nvidia-driver-latest-dkms
+yum install -y cuda-toolkit-12-3
+
+rapids-logger "Install GCC"
+yum install -y centos-release-scl
+yum install -y devtoolset-9
+
+source scl_source enable devtoolset-9
```

Comment:
Can you give some context on this? We're using `devtoolset-11` in https://github.com/rapidsai/ci-imgs/blob/main/ci-wheel.Dockerfile

Reply:
This was needed when we used previous images that didn't have a recent enough gcc, and has been removed.